backend/internal/database/book.go

```diff
@@ -129,6 +130,13 @@ func (db *BookDatabase) InsertBook(ctx context.Context,
 		return nil, errors.Wrap(err, "error on INSERT INTO books in InsertBook")
 	}
 
+	query = "INSERT INTO book_analytics (id) VALUES($1);"
```

Comment:
@dacruz21 Shouldn't this take care of updating book_analytics whenever new books are added?